analysis/cnvs/parse_prep_normalize_nch_cnvs.ipynb

```diff
@@ -64,14 +85,66 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "### download underlying data set of microarray CNV results; if file has not been downloaded, go to download_data_for_cnv_analysis_data.ipynb notebook\n",
-    "da = pd.read_csv(\"cnv_data/NCH-microarray-CNVs.csv\")"
+    "pd.set_option(\"display.max_columns\", 100)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
+   "outputs": [],
+   "source": [
+    "MANUSCRIPT_S3_URL = \"https://nch-igm-wagner-lab-public.s3.us-east-2.amazonaws.com/variation-normalizer-manuscript/2024\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download NCH microarray CNV data from s3 bucket\n",
+    "\n",
+    "from pathlib import Path\n",
+    "import requests\n",
+    "\n",
+    "def download_s3(url: str, outfile_path: Path) -> None:\n",
+    "    \"\"\"Download objects from public s3 bucket\n",
+    "\n",
+    "    :param url: URL for file in s3 bucket\n",
+    "    :param outfile_path: Path where file should be saved\n",
+    "    \"\"\"\n",
+    "    with requests.get(url, stream=True) as r:\n",
+    "        r.raise_for_status()\n",
+    "        with open(outfile_path, \"wb\") as h:\n",
+    "            for chunk in r.iter_content(chunk_size=8192):\n",
+    "                if chunk:\n",
+    "                    h.write(chunk)\n",
+    "path = \"cnv_data\"\n",
+    "Path(path).mkdir(exist_ok=True)\n",
+    "\n",
+    "for fn in [\n",
+    "    \"NCH-microarray-CNVs.csv\",\n",
+    "]:\n",
```

Comment:
We don't need to have this in a list anymore if this is the only file that's being downloaded